task-13/src/main/java/com/epam/winter_java_lab/task_13/exception/MessageNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.winter_java_lab.task_13.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
+public class MessageNotFoundException extends RuntimeException {
```

Comment:
Why do we need different not found exceptions for our entities? could we have only one NotFoundException class?